src/kili/gateways/kili_api_gateway/issue/operations.py

```diff
@@ -0,0 +1,33 @@
+"""GraphQL Issues operations."""
+
+GQL_CREATE_ISSUES = """
+mutation createIssues(
+    $issues: [IssueToCreate!]!
+    $where: AssetWhere!
+) {
+  data: createIssues(
+    issues: $issues
+    where: $where
+  ) {
+    id
+  }
+}
+"""
+
+
+def get_gql_issues_query(fragment):
+    """Return the GraphQL issues query."""
+    return f"""
+        query issues($where: IssueWhere!, $first: PageSize!, $skip: Int!) {{
+            data: issues(where: $where, first: $first, skip: $skip) {{
+                {fragment}
+            }}
+        }}
+        """
```

Comment:
this is not used currently, but kept for later?

Reply:
Ah yeah it would be for later. It is because I tried to implement the issue query. i will delete it for the moment